proto/prehog/v1alpha/teleport.proto

```diff
@@ -361,6 +361,24 @@ message UICreateNewRoleViewDocumentationClickEvent {
   string user_name = 1;
 }
 
+// UICallToActionClickEvent is a click in a Teleport Web UI's CTA
```

Comment:
These are unrelated changes: the types were newly introduced to Cloud in https://github.com/gravitational/cloud/pull/4326 , and will merged to Teleport master in https://github.com/gravitational/teleport/pull/25325 . Once that happens, we should merge master into `dev-ai` and these changes will "disappear".

Reply:
@justinas I updated the `dev-ai` with the latest master. Unfornetly, GH is not able to resolve the conflicts automatically, as all PRs are merged with squash which changes the commits IDs.